src/app/components/FatalErrorHandler/index.tsx

```diff
@@ -104,15 +104,16 @@ export function FatalErrorHandler({ children }: Props) {
             }
           />
         </Box>
-        {isExtension && (
-          <Box align={isMobile ? 'stretch' : 'end'} margin={{ bottom: isMobile ? 'xlarge' : 'large' }}>
-            <BrandButton
-              onClick={() => (window as any).chrome?.runtime?.reload()}
-              icon={<Refresh />}
-              label={t('fatalError.reloadExtension', 'Reload extension')}
-            />
-          </Box>
-        )}
+        <Box align={isMobile ? 'stretch' : 'end'} margin={{ bottom: isMobile ? 'xlarge' : 'large' }}>
+          <BrandButton
+            onClick={() => {
+              ;(window as any).chrome?.runtime?.reload()
```

Comment:
Any reason for this semicolon here?

Reply:
Prettier automatically adds it. It's meant to prevent concatenating this line with previous one

e.g. if lines were switched and it didn't add it:
```ts
window.location.reload()
(window as any).chrome?.runtime?.reload()
```
executes as `window.location.reload()(window).chrome?.runtime?.reload()`